index.html

```diff
@@ -543,7 +543,7 @@ <h4 id="return-values">Return Value</h4>
     <p>A request initiated by sendBeacon is subject to following properties:</p>
     <ul>
       <li>If the payload is one of <code>application/x-www-form-urlencoded</code>, <code>multipart/form-data</code>, or <code>text/plain</code>, then the request is a simple request that does not require an additional CORS-preflight; same as scripted form-post or XHR/fetch.</li>
```

Comment:
Right, ok.. with you now. How about ~~

- If there request does not contain a payload, then the request mode is `no-cors`; similar to an image beacon. Otherwise:
  - If the request Content-Type is a CORS-safelisted request-header, then the request is a simple request that does not require an additional CORS-preflight; similar to XHR/fetch.
  - Otherwise, a CORS preflight is made and the server needs to first allow such requests by returning the appropriate set of CORS headers (Access-Control-Allow-Credentials, Access-Control-Allow-Origin, Access-Control-Allow-Headers); similar to XHR/fetch.